internal/rpc/user_devices.go

```diff
@@ -451,9 +451,11 @@ func (s *userDeviceRPCServer) deviceModelToAPI(ud *models.UserDevice) *pb.UserDe
 		if sd := ud.R.VehicleTokenSyntheticDevice; sd != nil && !sd.TokenID.IsZero() {
 			stk, _ := sd.TokenID.Uint64()
 			iTkID, _ := sd.IntegrationTokenID.Uint64()
+			wc := sd.WalletChildNumber
```

Comment:
@JamesReate , is it all that is required for mapping?

Reply:
looks like it yeah